src/docs/Foundations/Typography/HeadingStyles/index.mdx

```diff
@@ -1,49 +1,92 @@
+### Heading Medium
 
-Heading styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+These are the default values for your heading styles for every breakpoint.
 
-## Headings styles define three properties of a text :
+| Breakpoint              | Heading size    |
+| ----------------------- | --------------  |
+| `S` (320px and up)      | 28px (1.75mu)   |
+| `M` - (680px and up)    | 34px (2.125mu)  |
+| `L` - (1024px and up)   | 41px (2.5625mu) |
+| `XL` - (1280px and up)  | 41px (2.5625mu) |
+| `XXL` - (1920px and up) | 41px (2.5625mu) |
 
-- Font-size _(that may vary depending on the viewport)_
-- Line-height _(that are optimised for readability and accessibility)_
-- Font-weight _(regular or semi-bold)_
+_heading medium preview_
 
-## Weight and line-height variations :
+## Specific needs
 
-Each body pattern comes with a modifier to set the font-weight to **'semi bold'** and a modifier to use an alternative **line-height**.
+This heading styles system still allows you to answer specific needs depending on the content of your pages or your patterns.
 
 ### Heading Small
 
-**Font-size :**  
-Viewport < M (680px) : 23px (1.4375mu)  
-Viewport >= M (680px) : 28px (1.75mu)
-
-<Preview path="typographyHeadingSmall" nude />
+These heading styles sizes are the smallest sizes you can use for your titles.
 
-### Heading Medium
+| Breakpoint              | Heading size   |
+| ----------------------- | -------------- |
+| `S` (320px and up)      | 23px (1.75mu)  |
+| `M` - (680px and up)    | 28px (1.75mu)  |
+| `L` - (1024px and up)   | 34px (2.125mu) |
+| `XL` - (1280px and up)  | 34px (2.125mu) |
+| `XXL` - (1920px and up) | 34px (2.125mu) |
 
-**Font-size :**  
-Viewport < M (680px) : 28px (1.75mu)  
-Viewport >= M (680px) : 34px (2.125mu )
+_heading small preview_
 
-<Preview path="typographyHeadingMedium" nude />
+<Highlight theme="warning" title="Be aware">
+If you use smaller values you will loose hierarchy with the body of your page or pattern.
+</Highlight>
 
 ### Heading Large
 
-**Font-size :**  
-Viewport < M (680px) : 34px (2.125mu)  
-Viewport >= M (680px) : 41px (2.5625mu)
+Mozaic provides large heading styles when you need to make some titles stand out. You can find these cases on landing pages for example.
+
+| Breakpoint              | Heading size    |
+| ----------------------- | --------------  |
+| `S` (320px and up)      | 34px (2.125mu)  |
+| `M` - (680px and up)    | 41px (2.5625mu) |
+| `L` - (1024px and up)   | 49px (3.0625mu) |
+| `XL` - (1280px and up)  | 49px (3.0625mu) |
+| `XXL` - (1920px and up) | 49px (3.0625mu) |
```

Comment:
It may not be necessary to specify these breakpoints, since there is no change from `L`